### PR TITLE
Display pull request information

### LINKED ISSUE #7: Show pull request information
It'd be nice if we could display some info about the pull request the user is searching. Using Github's API we can fetch a bunch of info about each PR. For example

```bash
❯ curl https://api.github.com/repos/servo/servo/pulls/19912
```
among other info, it gives the url, the status, the title, the owner and the creation date. It would be nice if we could show this info and a link to the original PR.

Github API reference: https://developer.github.com/v3/pulls/#get-a-single-pull-request

### PR DESCRIPTION
**Pull request by ferjm**
Fixes #7

### CODE REVIEW

**Comment by ferjm** [comment]
It looks like this:

screenshot

File: app/scripts/github.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+/**
+ * Partial Github REST API client.
+ * Reference: https://developer.github.com/v3
+ */
+export class GithubClient {
+  /**
+   * Initializes the client with the API endpoint URL.
+   */
+  constructor() {
+    this._api = GITHUB_API_URL;
```
Comment (CYBAI) [inline app/scripts/github.js]: nit: How about changing it like `this._apiUrl` to make it more clear that it's the API url?

File: app/scripts/ui.js [hunk 2]
```diff
@@ -49,12 +54,28 @@ const UI = {
...
+
+    this._github = new GithubClient();
+    this._github.fetchPullRequestInfo(id)
+    .then(pullRequest => {
+      if (!pullRequest) {
+        /// XXX pull request not found error.
+        return this.cancel();
+      }
+      this._showPullRequest(pullRequest);
+    });
```
Comment (CYBAI) [inline app/scripts/ui.js]: Should we `.catch` here to handle unexpected promise exception?

Comment (ferjm) [reply app/scripts/ui.js]: Yeah, good catch ;)

**Pull request merged**